CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## UNRELEASED
 
+### Adds
+
+* Wepback cache for build performance in development and production mode.
```

Comment:
Should mention the cache clearing task for edge bases where it is not automatically detected that the cache needs to be cleared.